test/ejabberd_tests/tests/privacy_SUITE.erl

```diff
@@ -395,7 +416,8 @@ block_group_message(Config) ->
         %% Alice should NOT receive message
         escalus_client:send(Bob, escalus_stanza:chat_to(Alice, <<"Hi, blocked group!">>)),
         timer:sleep(?SLEEP_TIME),
-        escalus_assert:has_no_stanzas(Alice)
+        escalus_assert:has_no_stanzas(Alice),
+        privacy_helper:gets_error(Bob, su)
```

Comment:
Ok, I see it now. Actually I don't like it. I'd rather make a helper function with more meaningful name, or use the full binary instead of `su` or `na`.